Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -0,0 +1,875 @@
+            # Table 3
+            human_readable += tableToMarkdown(t=ec_other,
+                                              name='Pulses General')
+
+    return_outputs(readable_output=human_readable,
+                   outputs=ec,
+                   raw_response=raw)
+
+
+# url command
+def url():
+    api_path = {
+        'source': 'indicators',
+        'command': 'url',
+        'value': demisto.args().get('url'),
+        'section': 'general'
+    }
+
+    # raw_data
+    raw = http_request(source=api_path['source'],
+                       command=api_path['command'],
+                       value=api_path['value'],
+                       section=api_path['section'])
+    ec = {
+        'URL': {
+            'Data': raw.get('indicator')
+        },
+        'AlienVaultOTX': {
+            'URL': {
+                'Hostname': raw.get('hostname'),
+                'Domain': raw.get('domain'),
+                'Alexa': raw.get('alexa'),
+                'Whois': raw.get('whois')
+            }
+        },
+        'DBotScore': {
+            'Indicator': raw.get('indicator'),
+            'Score': dbot_score(raw.get('pulse_info')),
+            'Type': 'URL',
+            'Vendor': 'AlienVault OTX'
+        }
+    }
+
+    ec = remove_none(ec)
+    return raw, ec
+
+
+def url_command():
+    raw, ec = url()
+
+    human_readable = ''
+    # Table 1
+    if ec.get('URL'):
+        human_readable += tableToMarkdown(t=ec.get('URL'),
+                                          name='URL')
+    # Table 2
+    if ec.get('AlienVaultOTX'):
+        if ec.get('URL'):
+            human_readable += tableToMarkdown(t=ec.get('AlienVaultOTX').get('URL'),
+                                              name='URL AlienVaultOTX')
+    # Table 3
+    if ec.get('DBotScore'):
+        human_readable += tableToMarkdown(t=ec.get('DBotScore'),
+                                          name='DBotScore')
+
+    return_outputs(readable_output=human_readable,
+                   outputs=ec,
+                   raw_response=raw)
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+commands = {
+    'test-module': test_module,
+    'ip': ip_command,
+    'domain': domain_command,
+    'alienvault-search-ipv6': alienvault_search_ipv6_command,
+    'alienvault-search-hostname': alienvault_search_hostname_command,
+    'file': file_command,
+    'alienvault-search-cve': alienvault_search_cve_command,
+    'alienvault-get-related-urls-by-indicator': alienvault_get_related_urls_by_indicator_command,
+    'alienvault-get-related-hashes-by-indicator': alienvault_get_related_hashes_by_indicator_command,
+    'alienvault-get-passive-dns-data-by-indicator': alienvault_get_passive_dns_data_by_indicator_command,
+    'alienvault-search-pulses': alienvault_search_pulses_command,
+    'alienvault-get-pulse-details': alienvault_get_pulse_details_command,
+    'url': url_command
```

Comment:
for more command enhancements it will not show this line as changed
```suggestion
    'url': url_command,
```

Reply:
👍